packages/compass-collection/src/components/collection-header/collection-header.spec.tsx

```diff
@@ -68,6 +69,7 @@ describe('CollectionHeader [Component]', function () {
     beforeEach(function () {
       render(
         <CollectionHeader
+          insights={[]}
```

Comment:
If something is an optional prop a more common approach is to actually mark it as optional key, not just as a union with `undefined`

Reply:
No, this is in addition to focus mode header. Feel free to keep it here, I would just want to highlight two things that this way of implementing it causes:

1) If something is truly optional in a component I would really suggest making it an optional key next time and not an empty something, the benefits of this are much nicer component interface, this is highlighed by how you had to add an empty array to every test case in this file even though this is just not used at all in the render
2) The value is memoized in the parent and you currently know about this, but without knowing that you need to memoize something to avoid unneccessary re-renders this interface makes it very easy to misuse it, causing unneccesary updates